### PR TITLE
Log pipeline related operations done by the configurator

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Add logging to XR pipeline/package installation related operations so that the user can know the exact operations the configurator performed to the project.

### CODE REVIEW

File: Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs [hunk 1]
```diff
@@ -368,6 +372,7 @@ private void RenderSelectXRSDKPlugin()
             {
 
 #if UNITY_2020_2_OR_NEWER
+                Debug.Log(MRTKConfiguratorLogPrefix + " Installing the Unity OpenXR Plugin. Operation performed is equivalent to installing the OpenXR plugin manually via Window -> Package Manager -> Packages: Unity Registry.");
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs:375]: For this one (and any others that are triggered by a button press), I wonder if we want to either

1. Make sure the button's label lets the user know it's about to install something when pressed
2. Pop-up a secondary dialog when the button is present to confirm the user definitely wants to install the package

Comment (MaxWang-MS) [reply Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs:375]: I'd say let's go with option 1.

Comment (MaxWang-MS) [reply Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs:375]: Just made the update. Also I want to mention that I don't think the installation of packages performed by the configurator has a high risk of user complaining as the packages to be installed are all Unity packages.

File: Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs [hunk 2]
```diff
@@ -364,22 +368,24 @@ private void RenderSelectXRSDKPlugin()
...
                 AddUPMPackage("com.unity.xr.openxr");
                 CurrentStage = ConfigurationStage.InstallOpenXR;
 #endif // UNITY_2020_2_OR_NEWER
                 Repaint();
             }
             CreateSpace(15);
             if (GUILayout.Button("\n<b>Built-in Unity plugins (non-OpenXR)</b><size=4>\n\n</size>"
-            + "Choose this if your application needs to support platforms beyond just HoloLens 2 and Windows Mixed Reality headsets (e.g. Oculus/Magic Leap headsets which are currently not supported via OpenXR).\n", multiLineButtonStyle))
+            + "Choose this if your application needs to support platforms beyond just HoloLens 2 and Windows Mixed Reality headsets (e.g. Oculus/Magic Leap headsets which are currently not fully supported via OpenXR). The Unity XR Management Plugin will be installed if not already.\n", multiLineButtonStyle))
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs:383]: nit, but I think Oculus headsets are now more properly supported by Unity in the latest release:

image

We still need to add support for the Oculus Touch controller via OpenXR in MRTK though

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs (hunk 2 above)
Comment (davidkline-ms) [inline Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs:383]:
```suggestion
            + "Choose this if your application needs to support platforms beyond HoloLens 2 and Windows Mixed Reality headsets (e.g. Oculus/Magic Leap headsets which are currently not fully supported via OpenXR). The Unity XR Management Plugin will be installed if not already.\n", multiLineButtonStyle))
```

**Review by davidkline-ms — APPROVED** [review]
suggested minor wording update, otherwise. looks good

**Pull request merged**